src/Mapping/Driver/DatabaseDriver.php

```diff
@@ -313,9 +317,15 @@ private function buildIndexes(ClassMetadata $metadata): void
                 continue;
             }
 
+            if (enum_exists(IndexType::class)) {
```

Comment:
I'd check for both the method and the enum.

Reply:
done